quickstarts/aws/newrelic-alert-cloudformation-resource/config.yaml

```diff
@@ -20,7 +20,7 @@ documentation:
   - name: Resource installation and configuration documentation
     description: |
       Installation and configuration instructions
```

Comment:
PLease mention comment "We are temporarily redirecting  to this [repository]( https://github.com/msummers-nr/newrelic-alerts-cloudformation). We will update soon"